### PR TITLE
Add grafana dashboard

### PR DESCRIPTION
**Pull request by dza89**
Signed-off-by: daan <daan.potter@nn-group.com>

**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

> /kind documentation

> /kind failing-test

/kind feature

> If this PR will release a new chart version please make sure to also uncomment the following line:

> /kind chart-release

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area falco-chart

/area falco-exporter-chart

> /area falcosidekick

**What this PR does / why we need it**:
Insert grafana dashboard into helm chart

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Checklist**

- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] CHANGELOG.md updated

### CODE REVIEW

File: falco-exporter/README.md [hunk 1]
```diff
@@ -73,7 +76,3 @@ helm install falco-exporter -f values.yaml falcosecurity/falco-exporter
 ```
 
 > **Tip**: You can use the default [values.yaml](values.yaml)
-
-## Using falco-exporter with Grafana
-
-See the [Grafana section](https://github.com/falcosecurity/falco-exporter#grafana) in the [falco-exporter](https://github.com/falcosecurity/falco-exporter) documentation.
```
Comment (leogr) [inline falco-exporter/README.md:79]: Could you document how to enable the dashboard and what requirements are needed to make it work?

Comment (dza89) [reply falco-exporter/README.md:79]: Isn't that literally done in the value part of the readme?

Comment (dza89) [reply falco-exporter/README.md:79]: Added the url over there

Comment (leogr) [reply falco-exporter/README.md:79]: Thank you, I will try this PR as soon as I can

Comment (leogr) [reply falco-exporter/README.md:79]: /assign

**Review by leogr — APPROVED** [review]

**Pull request merged**